app/routes/screens/racePredictor/components/CountDown.js

```diff
@@ -111,64 +106,16 @@ class CountDown extends Component {
 
   render() {
     const countDown = this.state;
-    let days;
-    if (countDown.days === 1) {
-      days = this.props.days.singular;
-    } else {
-      days = this.props.days.plural;
-    }
+
     return (
       <View style={styles.container}>
-        <Grid containerStyle={styles.grid}>
-          <Col size={25}>
-            <Row containerStyle={styles.rowTop}>
-              <Text style={styles.text}>
-                {CountDown.leadingZeros(countDown.days)}
-              </Text>
-            </Row>
-            <Row containerStyle={styles.rowLow}>
-              <Text style={styles.text}>
-                {days}
-              </Text>
-            </Row>
-          </Col>
-          <Col size={25}>
-            <Row containerStyle={styles.rowTop}>
-              <Text style={styles.text}>
-                {CountDown.leadingZeros(countDown.hours)}
-              </Text>
-            </Row>
-            <Row containerStyle={styles.rowLow}>
-              <Text style={styles.text}>
-                {this.props.hours}
-              </Text>
-            </Row>
-          </Col>
-          <Col size={25}>
-            <Row containerStyle={styles.rowTop}>
-              <Text style={styles.text}>
-                {CountDown.leadingZeros(countDown.min)}
-              </Text>
-            </Row>
-            <Row containerStyle={styles.rowLow}>
-              <Text style={styles.text}>
-                {this.props.minutes}
-              </Text>
-            </Row>
-          </Col>
-          <Col size={25}>
-            <Row containerStyle={styles.rowTop}>
-              <Text style={styles.text}>
-                {CountDown.leadingZeros(countDown.sec)}
-              </Text>
-            </Row>
-            <Row containerStyle={styles.rowLow}>
-              <Text style={styles.text}>
-                {this.props.seconds}
-              </Text>
-            </Row>
-          </Col>
-        </Grid>
+        <Text>{`${CountDown.leadingZeros(
+          countDown.days
```

Comment:
thats much better but it s hard to read.

try create const days = CountDown.leadingZeros(     countDown.days)
and ....
and then use them in the string template it should become easy to read